README.md

```diff
@@ -24,6 +24,8 @@
 OK so you've decided to redo your UI, don't worry, your UI will look as great as the screenshots in 5 simple steps! First of all, you will need to install [**card-mod**](https://github.com/thomasloven/lovelace-card-mod). It is a custom card available on [HACS](https://hacs.xyz) (the Home Assistant Community Store). Please read HACS's [documentation](https://hacs.xyz) and install it.
 
 ### 2. sun.sun
+As of version 0.114 of Home Assistant you are no longer required to use the ```sun.sun``` sensor to switch between dark and light mode **if** your device and browser support dark mode detection.
```

Comment:
Soft UI uses custom themes. Dark mode detection for custom themes isn't supported yet, so you should clarify this only works if you don't use a custom theme.

Reply:
Never mind, saw the part below. Cool feature! How about you link to that bottom part here?

Reply:
Done